quinn-proto/src/crypto.rs

```diff
@@ -390,12 +398,26 @@ pub struct ZeroRttCrypto {
     cipher: &'static aead::Algorithm,
 }
 
-#[derive(Clone)]
 pub struct CryptoContext {
     local: CryptoState,
     remote: CryptoState,
```

Comment:
As in the previous PR, I'd like to avoid storing the key bytes redundantly. This will probably require some refactoring, perhaps including doing away with `CryptoState`.